lib/bgsite_official_web/templates/admin/index.html.eex

```diff
@@ -0,0 +1 @@
+<%= def render(index.html)%>
```

Comment:
Thats is our html, no need for dynamic elm. yet, just put a `<h1> Testing Admin Template </h1>`

Reply:
I think also that ```index.html``` have to be in double quotes.